cmd/produce.go

```diff
@@ -30,6 +33,15 @@ Press Ctrl+D to exit.`,
 				}
 				defer producer.Close()
 
+				var schemaID uint32 = 0
```

Comment:
No need to explicitly set to 0
```suggestion
				var schemaID uint32
```